File: python/sortFunctions/bubble.py
import datetime

def bubble (vector, len):
    start = datetime.datetime.now()

    i = 0
    while (i < len):
        j = 0
        while(j < len - 1 - i):
            if (vector[j + 1] < vector[j]):
                aux = vector[j]
                vector[j] = vector[j + 1]
                vector[j + 1] = aux
            j += 1
        i += 1

    
    finish = datetime.datetime.now()
    time = finish - start
    print(f'Bubble {len} - time: {time.total_seconds() * 1000}')

File: js/sortFunctions/bubble.js
const bubble = (vector, len) => {
    const start = new Date().getTime();

    let aux, i, j
    for (i = 0; i < len; i++) {
        for (j = 0; j < len - 1 - i; j++) {
            if (vector[j + 1] < vector[j]) {
                aux = vector[j]
                vector[j] = vector[j + 1]
                vector[j + 1] = aux
            }
        }
    }
    
    const finish = new Date().getTime()
    const time = finish - start
    console.log(`Bubble ${len} - time: `, time)
    if (len === 100000) {
        console.log('-----------------')
    }
}

module.exports = {
    bubble
}